cli/azd/internal/appdetect/appdetect.go

```diff
@@ -153,6 +153,14 @@ func (a AzureDepEventHubs) ResourceDisplay() string {
 	return "Azure Event Hubs"
 }
 
+type AzureDepEventHubsForKafka struct {
```

Comment:
How about reuse `AzureDepEventHubs` instead of creating a new `AzureDepEventHubsForKafka`?

Reply:
Updated to reuse the `AzureDepEventHubs`. 
BTW, after synced with @saragluna, we can assume the customer use EventHub **OR** Kafka, don't have to support both. Because the Azure EventHub is actually the replacement of Kafka on Azure Cloud.